tajo-core/src/test/java/org/apache/tajo/storage/TestRowFile.java

```diff
@@ -128,8 +128,8 @@ public void test() throws IOException {
       scanner = new RowFile.RowFileScanner(conf, schema, meta, fragment);
       scanner.init();
       while ((tuple=scanner.next()) != null) {
-        if (!idSet.remove(tuple.get(0).asInt4())) {
-          System.out.println("duplicated! " + tuple.get(0).asInt4());
+        if (!idSet.remove(tuple.getInt4(0))) {
+          System.out.println("duplicated! " + tuple.getInt4(0));
```

Comment:
Even though you didn't change this line, it looks unnecessary. Would you remove it?